styles/sass/sf-icons.scss

```diff
@@ -166,8 +166,10 @@
 
 [class^="svg-icon-"], [class*="svg-icon-"] {
     display: inline-block;
-    width:20px;
-    height: 20px;
+    :is(.svg-icon-illustration) {
```

Comment:
Why the change to limit width/height to just `svg-icon-illustration`?
Looking at the history of this file, the width and height for `svg-icon` has been `20px` for 9+ years. Does this break anything?

Reply:
Yes it does, we have new components for SVG illustrations that fritz introduced. Because of that if I don't have this condition the alignment of them breaks.

Reply:
It's not clear to me too. You should either add CSS or if existing CSS conflicts with your markup, you should use `:not` selector to make existing CSS not apply to your new markup. It seems to be that you're instead changing behavior for all existing usages that are not yours. Or am I not correct?